DOCS.md

```diff
@@ -69,270 +70,325 @@ In a Sponsored Ad Break flow, once the user hits a mid-roll break with a true[X]
+### Handling Ad Elimination
 
-Another important event to listen for is `AD_FREE_POD`. This signifies that the user has earned a credit with true[X] and, once the renderer signals it is complete, all linear video ads remaining in the current pod should be skipped. It's important to note that the player should not immediately resume playback once receiving this event -- rather it should note that it was fired and continue to wait for an `AD_COMPLETED` event.
+Skipping video ads is completely the responsibility of the app code. The SSAI API should provide enough information for the app to determine where the current pod end-point is, and the app, when appropriate, should fast-forward directly to this point when resuming playback.
 
-The last major event to fire will be an `AD_COMPLETED` event. This will signal to the app that the renderer has finished showing the true[X] unit to the user. The user may have opted out of true[X], let the choice card countdown expire, or may have completed a true[X] ad unit and earned an ad-free credit (an `AD_FREE_POD` event would have already been fired in the last case). At this point, the app should check if an `AD_FREE_POD` event has fired -- if so, the user should be fast-forwarded past all remaining linear video ads and playback should resume. If not, playback still should resume, but without fast-forwarding so the user receives the remaining payload of video ads.
 
+## TruexAdRenderer tvOS API
 
-### Handling Ad Elimination
+This is an outline of public `TruexAdRenderer` methods and `TruexAdRendererDelegate` methods (referred to as events).
 
-Skipping video ads is completely the responsibility of the app code. The Uplynk Preplay API should provide enough information for the app to determine where the current pod end-point is, and, when appropriate, should fast-forward directly to this point when resuming playback.
 
+### Adding `TruexAdRenderer` to Your Project
 
-## TruexAdRenderer tvOS API
+The easiest way to add `TruexAdRenderer` is via CocoaPods. The renderer will be maintained and distributed on Github.
 
-This is an outline of public `TruexAdRenderer` methods and delegate methods (referred to as events).
+1. Add the following Podspec repositories to your Podfile:
+    ```ruby
+    source 'https://github.com/CocoaPods/Specs.git'
+    source 'https://github.com/socialvibe/cocoapod-specs.git'
+    source 'https://innovid-cocoapods:3bbe7445d58951a8f89c5e54c0ebfdd17039589e@github.com/Innovid/cocoapods-spec.git'
+    source 'https://github.com/YOU-i-Labs/YouiTVAdRenderer-CocoaPod.git'
+    ```
+1. Add the TruexAdRenderer pod to your target:
+    ```ruby
+    target 'MyApp' do
+        use_frameworks!
+        pod 'TruexAdRenderer', '~> 3.6'
+        # [...]
+    end
+    ```
 
+**Note:** `TruexAdRenderer` and some of its dependencies are delivered as frameworks -- not as static libraries. As such, please include the `use_frameworks!` line in your target definition.
```

Comment:
Perhaps we should add an extra note here about why the `source` links are required (because true[X] + Innovid + You.i are not available in the public specs repo).

I believe part of the rationale is that all specs in that repo are automatically under the MIT open source license: https://github.com/CocoaPods/Specs#license

Reply:
How's this:
```
1. Add the following Podspec repositories to your Podfile:
    ```ruby
    source 'https://github.com/CocoaPods/Specs.git'
    source 'https://github.com/socialvibe/cocoapod-specs.git'
    source 'https://github.com/Innovid/cocoapods-spec.git'
    source 'https://github.com/YOU-i-Labs/YouiTVAdRenderer-CocoaPod.git'
    ```
    **Note:** Two of `TruexAdRenderer`'s dependencies, from `Innovid` and `YOU-i-Labs`, are not available in the public specs repo.
```

Reply:
Makes sense!